src/installer/tests/TestUtils/Command.cs

```diff
@@ -196,7 +197,22 @@ public Command Start()
 
             ReportExecBegin();
 
-            Process.Start();
+            // Retry if we hit ETXTBSY due to Linux race
+            // https://github.com/dotnet/runtime/issues/58964
+            for (int i = 0; i < 3; i++)
+            {
+                try
+                {
+                    Process.Start();
+                    break;
+                }
+                catch (Win32Exception e) when (e.Message.Contains("Text file busy"))
```

Comment:
catch should consider `i` - there should be an iteration that lets the exception propagate out.